Cabal/doc/installing-packages.rst

```diff
@@ -716,37 +744,53 @@ Paths in the simple build system
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--libdir``                 | ``$prefix``                                                 | ``$prefix/lib``           |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--libsubdir`` (others)     | ``$pkgid\$compiler``                                        | ``$pkgid/$compiler``      |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--libexecdir``             | ``$prefix\$pkgid``                                          | ``$prefix/libexec``       |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--datadir`` (executable)   | ``$prefix``                                                 | ``$prefix/share``         |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--datadir`` (library)      | ``C:\Program Files\Haskell``                                | ``$prefix/share``         |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--datasubdir``             | ``$pkgid``                                                  | ``$pkgid``                |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--docdir``                 | ``$prefix\doc\$pkgid``                                      | ``$datadir/doc/$pkgid``   |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--sysconfdir``             | ``$prefix\etc``                                             | ``$prefix/etc``           |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--htmldir``                | ``$docdir\html``                                            | ``$docdir/html``          |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--program-prefix``         | (empty)                                                     | (empty)                   |
-+------------------------------+-------------------------------------------------------------+---------------------------+
-| ``--program-suffix``         | (empty)                                                     | (empty)                   |
-+------------------------------+-------------------------------------------------------------+---------------------------+
+.. list-table:: Default installation paths
```

Comment:
On smaller window it makes more sense to have the unix column first, as the windows column as one lengthy entry which shifts the unix column out of view otherwise